recipes/lensfun/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "lensfun" %}
+{% set version = "0.3.95" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 82c29c833c1604c48ca3ab8a35e86b7189b8effac1b1476095c0529afb702808
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - pkg-config
+  host:
+    - doxygen
+    - glib
+    - libpng
+  run:
+    - glib
+    - libpng
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib64/liblensfun.so  # [linux]
+    - test -f ${PREFIX}/include/lensfun/lensfun.h  # [unix]
```

Comment:
Since you are shipping a library that can be linked against by some downstream package, you may want to add a run_export. See the conda build documentation. This would go in the build section.

Reply:
@beckermr would the run export here be liblensfun? I am unfamiliar with this...thanks!

Reply:
Yeah you would do something like

```yaml
build:
  run_exports:
    - {{ pin_subpackage('lensfun', exact=True) }}
```

see the docs here: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements